bin/expedius-install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+##----------------------------------------------------------------------
+## Expedius Install script
+##
+## Author: Dennis Warren @ Colcamex Resources
+## Date: February 2012
+## Updated: January 2020
+## This script is used to install Expedius Lab Auto Downloader onto
+## a server environment that supports Oscar EMR version 15*
+##----------------------------------------------------------------------
+
+# init variables
+TOMCAT_PATH=\/usr\/local\/tomcat\/webapps
+ADMIN_EMAIL=none
+APP_DATA=\/var\/lib\/expedius
+EMAIL_SERVICE=no
+EXPEDIUS_USERNAME=admin
+EXPEDIUS_PASSWORD=3xp3dius#1
+OSCAR_USERNAME=expedius
+OSCAR_PASSWORD=$(openssl rand -base64 8)
+OSCAR_NUMBER=999995
+OSCAR_PIN=1234
+OSCAR_DN=tomcat_oscar:8080
+OSCAR_CONTEXT=oscar
+ADMIN_EMAIL=root@localhost.com
+
+
+echo "Setting Expedius configuration properties"
+
+# get administrators email address.
```

Comment:
We can include an email service container, or config if desired. Just a note for future.

Reply:
Yes. This is generally a good idea for the future.  No rush for the Expedius install. It's not a feature in high demand.